recipes/existdb/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "4.3.1" %}
+
+package:
+  name: existdb
+  version: {{ version }}
+
+source:
+  fn: existdb-{{ version }}.jar
+  url: https://bintray.com/existdb/releases/download_file?file_path=eXist-db-setup-{{ version }}.jar
+  sha256: 81007980d821e13e7b705412c0d5306bf58dc00649eb092c51e5add54a19515d
+
+build:
+  number: 0
+  script: 
+    - java -jar existdb-{{ version }}.jar -console -options ${RECIPE_DIR}/answers.txt
+
+requirements:
+  build:
+    - openjdk
+  run:
+    - openjdk
+
+test:
+  commands:
+    - test -f ${PREFIX}/bin/run.sh          # [unix]
+    - test -f ${PREFIX}/bin/shutdown.sh     # [unix]
+    - test -f ${PREFIX}/bin/run.bat         # [win]
+    - test -f ${PREFIX}/bin/shutdown.bat    # [win]
```

Comment:
Wow! Those are rather general names!

Reply:
I totally agree, maybe would be good to create a folder inside of the bin

Reply:
I think that is up to you / the users.